site/src/pages/TemplatePage/DAUCharts.tsx

```diff
@@ -0,0 +1,108 @@
+import useTheme from "@material-ui/styles/useTheme"
+
+import { Theme } from "@material-ui/core/styles"
+import {
+  CategoryScale,
+  Chart as ChartJS,
+  ChartOptions,
+  defaults,
+  Legend,
+  LinearScale,
+  LineElement,
+  PointElement,
+  Title,
+  Tooltip,
+} from "chart.js"
+import { Stack } from "components/Stack/Stack"
+import { HelpTooltip, HelpTooltipText, HelpTooltipTitle } from "components/Tooltips/HelpTooltip"
+import { WorkspaceSection } from "components/WorkspaceSection/WorkspaceSection"
+import moment from "moment"
+import { FC } from "react"
+import { Line } from "react-chartjs-2"
+import * as TypesGen from "../../api/typesGenerated"
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip, Legend)
+
+export interface DAUChartProps {
+  templateMetricsData: TypesGen.TemplateDAUsResponse
+}
+export const Language = {
+  loadingText: "DAU stats are loading. Check back later.",
+  chartTitle: "Daily Active Users",
+}
```

Comment:
I think now that we have internationalization, we're not using this pattern anymore and instead using the `t()` and then referencing strings from a `json` file. There should be some examples in the codebase but if you want specifics, let me know!

Reply:
Slightly off-topic, but why are we pursuing internationalization? That seems like a feature that is 3+ years out.

Reply:
Hey @ammario I was on vacation when you asked this, but I wanted to follow up. There are a couple reasons we're using this pattern, and some are outlined below:

1. It's helpful to have all strings colocated. We can be better about using consistent language, and we can re-use strings more easily.
2. Translation libraries have features like string interpolation or pluralization built into them, which helps us avoid a bunch of boilerplate JSX, keeping our component files lightweight.
3. We've actually had an internationalization solution in our app since I started, but prior to using i18n, we were using `Language` objects which is a problematic pattern for several reasons -you can read more in our [FE Variety notes](https://www.notion.so/coderhq/2022-07-22-28f0f1432c394f4b962853e7063993ef) if you're interested
4. Even if translation is a long way off, it's best to establish a pattern before the app grows large.

Anyway, let me know if you want to chat about it further!